doc/source/docs/api/geodataframe.rst

```diff
@@ -0,0 +1,87 @@
+============
+GeoDataFrame
+============
+.. currentmodule:: geopandas
```

Comment:
The path of those files can also be `/docs/reference/..` instead of `/docs/api/..` ? (since in the "index" is called reference)